packages/upset/src/components/Columns/Attribute/AttributeBar.tsx

```diff
@@ -48,17 +50,48 @@ export const AttributeBar: FC<Props> = ({ attribute, summary, row }) => {
   const items = useRecoilValue(itemsAtom);
   const values = getValuesFromRow(row, attribute, items);
 
+  const attributePlots = useRecoilValue(attributePlotsSelector);
+
   if (typeof summary !== 'number' && (summary.max === undefined || summary.min === undefined || summary.first === undefined || summary.third === undefined || summary.median === undefined)) {
     return null;
   }
 
+  /*
+   * Get the attribute plot to render based on the selected attribute plot type
+   * @returns {JSX.Element} The JSX element of the attribute
+   */
+  function getAttributePlotToRender() {
+    // for every entry in attributePlotType, if the attribute matches the current attribute, return the corresponding plot
+    // eslint-disable-next-line consistent-return
```

Comment:
Document why you need to disable `consistent-return`; this function shouldn't return `undefined` and if the linter thinks it can you should explain why it's wrong. Or explictly add `undefined` to the return type of the function.